tests/Aspire.Dashboard.Tests/Integration/Playwright/PlaywrightFixture.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Playwright;
+using Polly;
+using Polly.Retry;
+using Xunit;
+
+namespace Aspire.Dashboard.Tests.Integration.Playwright;
+
+public class PlaywrightFixture : IAsyncLifetime
+{
+    public IBrowser Browser { get; set; } = null!;
+
+    private IPlaywright _playwrightInstance = null!;
+    private ResiliencePipeline _resiliencePipeline = null!;
+
+    public async Task InitializeAsync()
+    {
+        var installExitCode = Microsoft.Playwright.Program.Main(new[] {"install"});
+        if (installExitCode != 0)
+        {
+            throw new PlaywrightException($"Playwright exited with code {installExitCode}");
+        }
+
+        var installDepsExitCode = Microsoft.Playwright.Program.Main(new[] {"install-deps"});
```

Comment:
On the linux (ubuntu) build machine `install-deps` will install a whole bunch of packages with `apt-get`. Because this is a shared machine, we want to avoid installing packages on it.

The existing infrastructure for installing playwright available is partially broken right now due to package installation issues (https://github.com/dotnet/aspire/issues/4623). And these tests are likely to run into the same issue randomly.

So, I would suggest not running the Playwright based tests on linux for now. You can add this to your test - `[ActiveIssue("https://github.com/dotnet/aspire/issues/4623", TestPlatforms.Linux)]`.

Going forward:
1. https://github.com/dotnet/aspire/pull/4689 - this will make the existing playwright targets reusable
2. Once https://github.com/dotnet/dnceng/issues/3122 is resolved, we can drop even this code.
Neither of these need to block your PR though.

Reply:
Also, could you please open an issue to track that the code here needs to be adjusted once https://github.com/dotnet/aspire/pull/4689 is merged, and https://github.com/dotnet/dnceng/issues/3122 is fixed?

Reply:
Usually we strongly recommend doing install and --with-deps from the outside. Since these require sudo and modify the system. These should not delay the test execution and skipping it on Linux seems like showing the drawbacks for it.

lets call it from the CI yaml instead?